### PR TITLE
Feature/add testing routes

### PR DESCRIPTION
**Pull request by LaimeJesus**
More refactors:
- add crud router and controller classes.
- add two endpoints `/test/db-clean` and `/test/db-seed`.

### CODE REVIEW

File: src/controllers/CrudController.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import GenericController from './GenericController'
+
+class CrudController extends GenericController {
+  constructor(service, logger) {
+    super(service, logger)
+  }
+
+  parseGET({ params: { id }}) {
```
Comment (FedeLochbaum) [inline src/controllers/CrudController.js:8]: `parseGET = ({ params: { id }}) => id`

File: src/routes/TestRouter.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import GenericRouter from "./GenericRouter"
+import TestController from "../controllers/TestController"
+
+class TestRouter extends GenericRouter {
+  constructor(router, logger) {
+    super(router, new TestController(), logger)
+  }
+
+  routes() {
+    const routes = super.routes()
```
Comment (FedeLochbaum) [inline src/routes/TestRouter.js:10]: ` routes = () => [...super.routes(), 
{ method: 'post', url: '/db-seed', message: 'seed' },
{ method: 'post', url: '/db-clean', message: 'clean' }]`

**Review by FedeLochbaum — APPROVED** [review]
Great! :) 

I left some comments about code styling

**Comment by LaimeJesus** [comment]
> Great! :)
> 
> I left some comments about code styling


Here's an extract explaining why I'd defined those methods using `function` instead of `arrow fn`:
```
// @TODO:
// - currently we can't override a parent class method
//   when it's defined as an arrow function
//   https://github.com/expressjs/express/issues/2963
```
This is a comment in the `TestController.js` file.

**Pull request merged**